pkg/governance-scripts/test/BaseCoordinator.test.ts

```diff
@@ -25,8 +23,9 @@ describe('BaseCoordinator', () => {
   sharedBeforeEach('deploy vault', async () => {
     vault = await Vault.create({ admin });
     if (!vault.authorizer) throw Error('Vault has no Authorizer');
+    if (!vault.authorizerAdaptor) throw Error('Vault has no AuthorizerAdaptor');
```

Comment:
Do we need this? It seems these values are not optional - we always have them. If so you can use the alternate format suggested in the other comment to condense all of this to a single line.

Reply:
A few of them have these, so I just followed the pattern. (Authorizer is also not optional.) Happy to remove them.